schema-engine/sql-introspection-tests/tests/postgres/mod.rs

```diff
@@ -309,3 +309,92 @@ async fn deferrable_stopgap(api: &mut TestApi) -> TestResult {
 
     Ok(())
 }
+
+#[test_connector(tags(Postgres), exclude(CockroachDb), preview_features("views"))]
+async fn commenting_stopgap(api: &mut TestApi) -> TestResult {
+    // https://www.notion.so/prismaio/Comments-ac89f872098e463183fd668a643f3ab8
+
+    let schema = indoc! {r#"
+        CREATE TABLE a (
+            id INT PRIMARY KEY,
+            val VARCHAR(20)
+        );
+
+        CREATE VIEW b AS SELECT val FROM a;
+
+        CREATE TYPE c AS ENUM ('a', 'b');
+
+        COMMENT ON TABLE a IS 'push';
+        COMMENT ON COLUMN a.val IS 'meow';
+        COMMENT ON VIEW b IS 'purr';
+        COMMENT ON TYPE c IS 'hiss';
+    "#};
+
+    api.raw_cmd(schema).await;
+
+    let expectation = expect![[r#"
+        generator client {
+          provider        = "prisma-client-js"
+          previewFeatures = ["views"]
+        }
+
+        datasource db {
+          provider = "postgresql"
+          url      = "env(TEST_DATABASE_URL)"
+        }
+
+        model a {
+          id  Int     @id
+          val String? @db.VarChar(20)
+        }
+
+        /// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+        view b {
+          val String? @db.VarChar(20)
+
+          @@ignore
+        }
+
+        enum c {
+          a
+          b
+        }
+    "#]];
+
+    api.expect_datamodel(&expectation).await;
+
+    let expectation = expect![[r#"
+        [
+          {
+            "code": 24,
+            "message": "The following views were ignored as they do not have a valid unique identifier or id. This is currently not supported by the Prisma Client. Please refer to the documentation on defining unique identifiers in views: https://pris.ly/d/view-identifiers",
+            "affected": [
+              {
+                "view": "b"
+              }
+            ]
+          },
+          {
+            "code": 36,
+            "message": "These objects have comments defined in the database, which is not yet fully supported. Read more: https://pris.ly/d/database-comments",
+            "affected": [
+              {
+                "type": "enum",
+                "name": "c"
+              },
+              {
+                "type": "model",
+                "name": "a"
+              },
+              {
+                "type": "field",
+                "name": "a.val"
+              }
+            ]
```

Comment:
Isn't this missing the comment on the view?

Reply:
Fixed in [`625d0a9` (#3873)](https://github.com/prisma/prisma-engines/pull/3873/commits/625d0a9660fe0c2f96af85749e8ccf5bc55aca74)

Reply:
Also: Comments on columns of views 🤯